packages/rx_bloc_cli/mason_templates/bricks/feature_qr_scanner/__brick__/lib/l10n/network_lookup/qr_code_lookup.dart

```diff
@@ -0,0 +1,33 @@
+{{> licence.dart }}
+
+import '../../assets.dart';
+import '../{{project_name}}_app_i18n.dart';
+import 'util.dart';
+
+class AppI18nQrLookup extends I18nFeatureQrLookup {
+  @override
+  String getString(String key, [Map<String, String>? placeholders]) {
+    return getFromNetwork('qrCode__', key, placeholders) ??
+        getFromBundle(AppI18n.locale?.languageCode)
+            .getString(key, placeholders)!;
+  }
+
+  I18nFeatureQr? bundledI18n;
+
+  I18nFeatureQr getFromBundle(String? languageCode) {
+    if (bundledI18n == null) {
+      switch (I18n.currentLocale?.languageCode) {
+        case 'bg':
+          bundledI18n = I18nFeatureQr(I18nFeatureQrLookup_bg());
+          break;
+        case 'en':
+          bundledI18n = I18nFeatureQr(I18nFeatureQrLookup_en());
+          break;
+        default:
+          bundledI18n = I18nFeatureQr(I18nFeatureQrLookup_en());
+          break;
```

Comment:
If you're repeating code for different cases in a `switch`, you can simply `fall-through` using the following syntax:

```dart
case 'bg':
  // code specific to `bg` case
case 'en':
default:
  // some code common to `en` and `default` cases
```